File: TransCoder_tokenized_test_set_functions/java/PRINT_REVERSE_STRING_REMOVING_VOWELS.java
static void replaceOriginal ( String s, int n ) {
  char r [ ] = new char [ n ];
  for ( int i = 0;
  i < n;
  i ++ ) {
    r [ i ] = s . charAt ( n - 1 - i );
    if ( s . charAt ( i ) != 'a' && s . charAt ( i ) != 'e' && s . charAt ( i ) != 'i' && s . charAt ( i ) != 'o' && s . charAt ( i ) != 'u' ) {
      System . out . print ( r [ i ] );
    }
  }
  System . out . println ( "" );
}

File: TransCoder_tokenized_test_set_functions/cpp/PRINT_REVERSE_STRING_REMOVING_VOWELS.cpp
void replaceOriginal ( string s, int n ) {
  string r ( n, ' ' );
  for ( int i = 0;
  i < n;
  i ++ ) {
    r [ i ] = s [ n - 1 - i ];
    if ( s [ i ] != 'a' && s [ i ] != 'e' && s [ i ] != 'i' && s [ i ] != 'o' && s [ i ] != 'u' ) {
      cout << r [ i ];
    }
  }
  cout << endl;
}